scripts/globals/mobskills/pollen.lua

```diff
@@ -24,6 +24,11 @@ mobskill_object.onMobWeaponSkill = function(target, mob, skill)
 
     potency = potency - math.random(0, potency/4)
 
+    if mob:getName() == "Beelzebub" then
```

Comment:
A few things you should be aware of here:
- Pollen's whole methodology is fubar, we know it, and am not gonna make you try and sort that out! This is fine for now. [I have an open issue the subject of fubar mobskills that uses pollen as its example.](https://github.com/LandSandBoat/server/issues/2163) Someday we'll revisit.
- One thing I am asking to change is that we normally match on pool instead of on name. `if mob:getPool() == 385 then` in this case.
- there's [another PR making a similar change for Demonic Tiphia,](https://github.com/LandSandBoat/server/pull/2098/files#diff-8a71732b9e62411bd52ad3fc44663ed88abe4d68c907791c5bd61f70f033a1e7R29) whichever PR merges 1st will have no problem, the one that merges 2nd will have to update and handle the edit conflict.

Reply:
Okay, can do! not to throw zach under the bus, but I used getName() under his advisory
I'll definitely use this from now on.

Reply:
thats ok, I'm driving that bus and I know where he lives :) 



j/k zach!